doc/intro.rst

```diff
@@ -1,58 +0,0 @@
-.. -*- coding: utf-8 -*-
-
-==============
```

Comment:
Is it worth breaking old links for this? I think we should add a redirect.

Reply:
I've searched a way to do it in the configuration file but I don't think it's possible. Do you happen to know how to do it ?

Reply:
No, I have never done this, but it should be possible. At least with a Sphinx extension.

Reply:
It seems it should be possible to do it with readthedoc conf directly but after 60+mn I just took an extension to do it.